pkg/kubecost/set.go

```diff
@@ -0,0 +1,13 @@
+package kubecost
+
+import "encoding"
+
+type Set interface {
```

Comment:
Probably want to be a bit more specific with a name here: `ETLSet`?

Reply:
Yeah I like ETLSet